### DISCUSSION TITLE
Is this only for Windows 11?

### DISCUSSION THREAD

**Discussion by parasiteoflife** [ramensoftware/windhawk#120] [Q&A] [answered]
I've only seen Windows 11 being named on the news posts and almost anywhere this application is named but it doesn't exactly says it only works with Windows 11, so my question is that. Also please clarify this in the repo and website.

**Comment by m417z** [comment, marked as answer]
Windhawk runs on Windows 7 and newer. Windows 7 is no longer supported by Microsoft, and support is likely to be dropped in one of the next Windhawk updates.

Regarding mods, some mods are Win11-only, but many aren't. For example, these mods support Win10:
* Disable grouping on the taskbar
* Taskbar Clock Customization

These mods support all versions, including Win7:
* Taskbar Volume Control
* Slick Window Arrangement

**Reply by parasiteoflife** [reply]
Thank you for the info, I'll move on to Windhawk then since I mostly use the disable grouping and taskbar volume
